examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -99,6 +99,67 @@ databases: []
 #      "*": "*"
 databaseResources: []
 
+################################################################
+# Values that must be provided if Windows Desktop access is enabled.
+################################################################
+
+# AD or non-AD configuration
+
+# (optional) Determines whether desktop sessions will show a user-selected wallpaper
+# vs a system-default, single-color wallpaper. For improved visual performance,
+# set this to false (its default value).
+#showWindowsDesktopWallpaper: false
+
+# (optional) ldap contains configuration keys used when connecting Teleport
+# to an Active Directory domain. This enables the discovery service for
+# Windows desktops belonging to an Active Directory domain configured for
+# Teleport access.
+#windowsDesktopLDAP:
```

Comment:
Is it common for us to use `camelCase` in values.yaml? It's a bit tricky to switch back and forth (`windowsDesktopLDAP` vs `ldap_ca_cert`, for example)

Reply:
Yes, camel case has been the approach (appResources,..).  Since we use `ldap` I changed this part to `windowsDesktopLDAP` as camelcase.  The other fields under the ldap config match.